core/lib/env_config/src/chain.rs

```diff
@@ -125,6 +125,8 @@ mod tests {
             CHAIN_STATE_KEEPER_SAVE_CALL_TRACES="false"
             CHAIN_STATE_KEEPER_UPLOAD_WITNESS_INPUTS_TO_GCS="false"
             CHAIN_STATE_KEEPER_ENUM_INDEX_MIGRATION_CHUNK_SIZE="2000"
+            CHAIN_STATE_KEEPER_VIRTUAL_BLOCKS_PER_MINIBLOCK="1"
```

Comment:
surprising - what's the reason for this to be in this PR?

Reply:
Yeah I was also surprised, but this test was failing for me locally. And if you look closely, these variables were indeed missing from the complete config. :shrug: